src/main/resources/application.properties

```diff
@@ -27,7 +27,7 @@ engine.skip=false
 rbac/mp-rest/url=https://ci.cloud.redhat.com
 
 # Notifications server
-notifications/mp-rest/url=http://custom-policies-notifications:8080
+notifications/mp-rest/url=http://custom-policies-notifications:8000
```

Comment:
How about we expose port 8080 like for other containers ?
I guess we just need to change:
https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/custom-policies/custom-policies-notifications.yml#L15

Reply:
I made a PR here: https://github.com/RedHatInsights/e2e-deploy/pull/1242
It that gets approved, I can already test I think

Reply:
I am not opposed to that. Having 8080 everywhere makes it harder locally when all services are deployed 'bare metal'. cc @burmanm

Reply:
Well, container can redirect ports so it doesn't matter - that PR didn't change the local port.

Reply:
So it's running on port 8080 now, we can change back if you prefer.

Reply:
What is the conclusion here? Do I change the default in here or not?

Reply:
it's listening on 8080 now